prow/README.md

```diff
@@ -1,124 +1,10 @@
-# ![Prow](logo_horizontal_solid.png)
+DOCUMENTATION DEPRECATION NOTICE: This file is deprecated. Please refer to the
+[new migrated
+location](https://github.com/kubernetes-sigs/prow/tree/main/site/content/en/docs/Legacy%20Snapshot/prow/README.md).
+Please do not edit this file; instead, make changes to the new location!
 
-Prow is a Kubernetes based CI/CD system. Jobs can be triggered by various types of events and report their status to many different services. In addition to job execution, Prow provides GitHub automation in the form of policy enforcement, chat-ops via `/foo` style commands, and automatic PR merging.
+The new location is served on the web at
+https://docs.prow.k8s.io/docs/legacy-snapshot/.
 
-See the [GoDoc](https://godoc.org/k8s.io/test-infra/prow) for library docs.
-Please note that these libraries are intended for use by prow only, and we do
-not make any attempt to preserve backwards compatibility.
+This file will be deleted on 2022-11-30.
```

Comment:
For followup (maybe add this as a comment to this file): when you sweep through and delete `prow/**/*.md`, I would recommend keeping a README.md file here that explains "code here, docs there" with links to the other repo / website.